src/GridView/CardGrid.js

```diff
@@ -6,25 +6,29 @@ import FlatButton from 'material-ui/FlatButton';
 import LoadingAnimation from './../Utils/LoadingAnimation';
 import { Row, Col } from 'react-grid-system';
 import { css } from 'aphrodite';
+import AppStyles from "./../AppStyles";
 import EntityInformation from './../DetailView/EntityInformation';
+import StatusDiagram from "./StatusDiagram";
 import config from './../config/config';
+import Utils from './../Utils/Utils';
 
 class CardGrid extends ConnectionComponent {
+
 	backgroundColor (status) {
-		if (status in config.status) {
-			return ({backgroundColor: config.status[status].color});
+		if (Utils.getStatus(status)) {
+			return ({backgroundColor: Utils.getStatus(status).color});
 		}
 		else {
-			return {backgroundColor: config.status["UNDEFINED"].color};
+			return {backgroundColor: Utils.getStatus("UNDEFINED").color};
```

Comment:
I think it would be better to define a UNDEFINE status and return it inside the Utils.getStatus() methid